docs/user/installation.rst

```diff
@@ -10,23 +10,28 @@ Express Install
 
 The fastest way to set up dependencies for garage is via running the setup script.
 
+Clone this repo and navigate to its directory.
+
+A MuJoCo key is required for installation.
+
 - On Linux, run the following:
 
 .. code-block:: bash
 
-    ./scripts/setup_linux.sh
+    ./scripts/setup_linux.sh --mjkey path-to-your-mjkey.txt
 
 - On Mac OS X, run the following:
 
 .. code-block:: bash
 
-    ./scripts/setup_osx.sh
+    ./scripts/setup_osx.sh --mjkey path-to-your-mjkey.txt
+
 
 The script sets up a conda environment, which is similar to :code:`virtualenv`. To start using it, run the following:
 
 .. code-block:: bash
 
-    source activate garage
+    conda activate garage
```

Comment:
Let's mention that `conda activate garage` only works for conda versions 4.4+, while those older than 4.4 should use `source activate <envname>`.

Reply:
I changed it back to source activate garage for simplicity. You think I should include both?

Reply:
Since we are eventually phasing out conda anyway, just keep `source` for simplicity.